sequelizeDatabase/sequelFunctions.js

```diff
@@ -1,350 +1,481 @@
-        defaults: {
-            Name: Name,
-            Comments: Comments}    
-    }).spread((user,createdBoolean) => {
-        return user;
-    }).catch(err => console.error('Error in insertPerson', err));
+  return models.Individuals.findOrCreate({
+    where: { LinkedInUrl: LinkedInUrl, FundID: FundID },
+    defaults: {
+      Name: Name,
+      Comments: Comments
+    }
+  })
+    .spread((user, createdBoolean) => {
+      return user;
+    })
+    .catch(err => console.error("Error in insertPerson", err));
 };
 
 insertCompany = (CompanyName, FundID) => {
-    return models.Companies.findOrCreate({where: {CompanyName: CompanyName, FundID: FundID}
-    }).spread((company, createdBoolean) => {
-        //console.log('Company Created: ', company);
-        return company;
-    }).catch(err => console.error('Error in insertCompany',err));
+  return models.Companies.findOrCreate({ where: { CompanyName: CompanyName, FundID: FundID } })
+    .spread((company, createdBoolean) => {
+      //console.log('Company Created: ', company);
+      return company;
+    })
+    .catch(err => console.error("Error in insertCompany", err));
 };
 
 insertOriginalFundPosition = (IndividualID, CompanyID, PositionName) => {
-    //TODO: Verify that the fundIDs in both of these match each other.
-    //      Otherwise, just use with Caution.
-    return models.OriginalFundPosition.create({//TODO: Maybe make this findOrCreate, check if Funds match.
-        IndividualID: IndividualID,
-        CompanyID: CompanyID,
-        PositionName: PositionName
-    }).then((originalFundPosition) =>{
-        return originalFundPosition;
-    }).catch(err => console.error('Error in "insertOriginalFundPosition", ', err));
+  //TODO: Verify that the fundIDs in both of these match each other.
+  //      Otherwise, just use with Caution.
+  return models.OriginalFundPosition.create({
+    //TODO: Maybe make this findOrCreate, check if Funds match.
+    IndividualID: IndividualID,
+    CompanyID: CompanyID,
+    PositionName: PositionName
+  })
+    .then(originalFundPosition => {
+      return originalFundPosition;
+    })
+    .catch(err => console.error('Error in "insertOriginalFundPosition", ', err));
 };
 
 insertEmployeeHistory = (UserID, IndividualID, CompanyID, PositionName, StartDate, EndDate) => {
-    return models.EmployeeHistory.create({
-        //TODO: Add a restriction so you can't have Individuals and companies with Mismatched FundID's.
-        UserID: UserID,
-        IndividualID: IndividualID,
-        CompanyID: CompanyID,
-        PositionName: PositionName,
-        StartDate: StartDate,
-        EndDate: EndDate
-    }).then((history) =>{
-        //console.log('History Added: ', history);
-        return history;
-    }).catch(err => console.error('Error in insertEmployeeHistory',err));
-}
+  return models.EmployeeHistory.create({
+    //TODO: Add a restriction so you can't have Individuals and companies with Mismatched FundID's.
```

Comment:
**ISSUE**: While we may be able to make it so this scenario never happens, it may be best to still add code to prevent such an issue from being possible.